jsapp/js/account/security/email/emailSection.component.tsx

```diff
@@ -112,6 +120,14 @@ export default function EmailSection() {
     }
   }
 
+  function userCanChangeEmail() {
```

Comment:
Can we make this a variable instead of a function? I think you could also use a ternary to shorten it up.